web/src/core/ports/OnyxiaApi/XOnyxia.ts

```diff
@@ -64,6 +64,7 @@ export type XOnyxiaContext = {
         defaultIpProtection: boolean | undefined;
         defaultNetworkPolicy: boolean | undefined;
         allowedURIPattern: string;
+        customValues: object | undefined;
```

Comment:
```suggestion
        customValues: Record<string, unknown>;
```
It would be easyer to have an empy object insted of undefined here to prevent runtime exceptions when using chart on an instance of onyxia that wouldn't have custom values defined.